install_config/aggregate_logging.adoc

```diff
@@ -899,6 +899,23 @@ xref:../install_config/install/advanced_install.adoc#configuring-node-host-label
 Fluentd uses `journald` as the system log source. These are log messages from
 the operating system, the container runtime, and OpenShift.
 
+The available container runtimes provide minimal information to identify the
+source of log messages. Log collection and normalization of logs may occur after
+a pod is deleted and there is no additional information to be retrieved from the
+API server (e.g., labels, annotations).
+
+If a log generating pod and namespace are deleted before the log collector finishes
```

Comment:
"If a pod with a given name and namespaces is deleted..."

Reply:
This entire statement is correct but I believe it needs to be broken into multiple sentences; it reads long.